Rakefile

```diff
@@ -11,7 +11,7 @@ task :console do
   if File.exist? config_file
     config = YAML.load File.read(config_file)
 
-    AWS.config(config['aws'])
+    Aws.config = config['aws']
```

Comment:
I don't use that task. I just ```rails console``` usually, but I think we definitely need a way to make Shoryuken work out of the box there without requiring the user to know a particular magic incantation to get it work.